src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -46,3 +86,152 @@ bbf_selectDumpableCast(CastInfo *cast)
 			 strcmp(tTypeInfo->dobj.name, "varchar") == 0))
 		cast->dobj.dump = DUMP_COMPONENT_NONE;
 }
+
+/*
+ * fixTsqlTableTypeDependency:
+ * Fixes following two types of dependency issues between T-SQL
+ * table-type and T-SQL MS-TVF/procedure:
+ * 1. T-SQL table-type has an INTERNAL dependency upon MS-TVF which
+ *    is right thing for drop but creates dependency loop during
+ *    pg_dump. Fix this by removing table-type's dependency on MS-TVF.
+ * 2. By default function gets dumped before the template table of T-SQL
+ *    table type(one of the datatype of function's arguments) which is
+ *    because there is no dependency between function and underlying
+ *    template table. Ideally function should have a dependency upon table
+ *    instead of table-type but it is fine in normal case but becomes
+ *    problematic during restore. Fix this by adding function's dependency
+ *    on template table.
+ */
+void
+fixTsqlTableTypeDependency(Archive *fout, DumpableObject *dobj, DumpableObject *refdobj, char deptype)
+{
+	FuncInfo  *funcInfo;
+	TypeInfo  *typeInfo;
+	TableInfo *tytable;
+	char	  *lanname;
```

Comment:
langname is more readable

Reply:
Looks like community prefers `lanname` :)